server/protocols/jmap-integration-testing/memory-jmap-integration-testing/src/test/java/org/apache/james/jmap/memory/MemoryForwardIntegrationTest.java

```diff
@@ -31,7 +33,14 @@
 
     @Override
     protected GuiceJamesServer createJmapServer() {
-        return memoryJmap.jmapServer();
+        return memoryJmap
+            .jmapServer(binder -> binder.bind(WebAdminConfiguration.class)
+                .toInstance(WebAdminConfiguration.builder()
```

Comment:
we could name this configuration and reuse it.
It's linked to the way we call webadmin in tests anyway, so factoring in this case looks a good strategy, WDYT ?

Reply:
Done